scripts/create_deprecated_module.lean

```diff
@@ -0,0 +1,175 @@
+/-
+Copyright (c) 2025 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+
+--import Mathlib.Init
+import Lean.Elab.Command
+import Mathlib.Tactic.Linter.DeprecatedModule
+-- a comment here
+
+/-!
+#  Create a deprecated module
+
+This file defines the lean script for creating a deprecated module.
+-/
+
+open Lean Elab Command
+
+namespace DeprecatedModule
+
+/--
+`getHeader fname fileContent keepTrailing` takes as input two strings and a `Bool`ean.
+It uses
+* `fname`, as the path of a file (which need not exist);
+* `fileContent`, as the content of `fname` (regardless of whether the file exists and what its
+  content it);
+* `keepTrailing` a boolean.
```

Comment:
this comment looks unfinished (but is explained in the next sentence). Can you reword this somehow?

Reply:
Done!